docs/source/faq/index.rst

```diff
@@ -80,24 +97,225 @@ of these popular operating systems from the past few years.
 We also provide :ref:`custom install instructions <alternative-builds>` to use
 FiftyOne on old-but-popular setups like Ubuntu 16.04 and Debian 9.
 
-Can you share a dataset with someone else?
-------------------------------------------
+.. _faq-share-dataset-export:
+
+Can I share a dataset with someone else?
+----------------------------------------
 
-You can easily :doc:`export a dataset </user_guide/export_datasets>` in one
-ine of code, zip it, and send it to someone else who can then
-:doc:`load it in a few lines of code. </user_guide/dataset_creation/datasets>`.
+Yes! Here's a couple options:
 
-Alternatively, you could launch a :ref:`remote session <remote-data>` of the
-FiftyOne App on your machine that another user can connect to from their local
-machine. This workflow does require that both users have the
+**Option 1: Export and share**
+
+You can easily :ref:`export a dataset <exporting-datasets>` in one line of
+code, zip it, and share the zip with your collaborator, who can then
+:ref:`load it in a few lines of code <loading-datasets-from-disk>`.
+
+**Option 2: Sharing a remote session**
+
+Alternatively, :ref:`see this FAQ <faq-multiple-sessions-same-dataset>` for
+instructions on launching a remote session and inviting collaborator(s) to
+connect to it from their local machines.
+
+.. _faq-brain-closed-source:
 
 Are the Brain methods open source?
 ----------------------------------
 
-No. Although the :ref:`core library <https://github.com/voxel51/fiftyone>` is
-open source and the :doc:`Brain methods </user_guide/brain>` are freely
-available for use for any commerical or non-commerical purposes, the Brain
-methods are closed source.
+No. Although the `core library <https://github.com/voxel51/fiftyone>`_ is open
+source and the :ref:`Brain methods <fiftyone-brain>` are freely available for
+use for any commerical or non-commerical purposes, the Brain methods are closed
+source.
 
-Check out the :doc:`Brain documentation </user_guide/brain>`` for detailed
+Check out the :ref:`Brain documentation <fiftyone-brain>` for detailed
 instructions on using the various Brain methods.
+
+.. _faq-multiple-sessions-same-dataset:
+
+Can I connect multiple App instances to the same dataset?
```

Comment:
I edited the various questions about "multiple connections" into the following three questions:

1) Multiple App instances
2) Connecting to multiple remote sessions
3) Serving multiple remote sessions from a machine

I found that the "programmatic" and "non-programmatic" FAQ questions were largely duplicating content from other FAQ questions that covered the same concepts. 

If you feel like "collaboration" is still an important keyword that you want to include in a question title, perhaps that answer should just link into the relevant other FAQ questions as defined here.

Reply:
I like the changes! We still have "collaborator" mentioned in the faq so searching it will bring up the FAQ which is good enough for me.